Makefile

```diff
@@ -1,52 +1,31 @@
-PROJECT_ROOT := src/
 VERSION = 1.0.7
 
 .DEFAULT_GOAL := all
 
-# The resulting binaries map to the subproject names
-BINARIES = \
-	abireport
+clean:
+	rm -f abireport abireport-*.tar.xz
+	rm -fr vendor
```

Comment:
Should probably remove `man/abireport.1` and `man/abireport.1.html`, too.

Reply:
Fair, will change.